src/commands/protocols/common/epoch-check-command.js

```diff
@@ -19,6 +20,11 @@ class EpochCheckCommand extends Command {
     }
 
     async execute(command) {
+        const operationId = uuidv4();
```

Comment:
Can we have here something like `this.operationIdService.generateId()`?

Reply:
Good point, I'll add this function